com.unity.shadergraph/Editor/Drawing/Views/StickyNote.cs

```diff
@@ -206,8 +219,20 @@ public enum Resizer
         }
 
         Dictionary<Resizer, VisualElement> m_Resizers = new Dictionary<Resizer, VisualElement>();
-    }
 
+        List<Manipulator> m_Manipulators = new List<Manipulator>();
```

Comment:
We prefer to declare members at the top of the file unless it's a very specific use case in a single function.